### ISSUE TITLE
Input field placeholder text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52593]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Windows 10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1. Launch VS Code.
2.Navigate to Activity Bar and select "View Debug "(Cntrl+Shift+D) button.
3 Navigate to Breakpoints section-->Add Function Breakpoint-->Input Field (Function to break on.)
4. Start Color Contrast Analyzer to determine the color contrast ratio of names of the toolbars.

**Actual:**
Color Contrast ratio for  placeholder is less than 4.5:1 i.e. Function to break on.

**Expected:**
Color Contrast ratio for placeholder should be >=4.5:1 i.e. Function to break on.

**Recommendations:**
Adjust color of that element to meet the minimum color contrast requirement. 
Or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low vision users will not clearly understand the placeholder text written on the screen.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference**
![accessibility a11y_vscode_viewdebug_colorcontrast](https://user-images.githubusercontent.com/39204909/41716567-7a886fdc-7574-11e8-90bf-671a0bad9e26.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@misolori assigning to you also.
Here's a code pointer https://github.com/Microsoft/vscode/blob/9ab41a4464e62739fa8850d2aa312de2f72eb20d/src/vs/base/browser/ui/inputbox/inputBox.ts#L205

**Comment by isidorn** [comment]
@misolori not sure if we can do something here. Pushing out to August, feel free to push back to July if you have some ideas regarding this

**Comment by misolori** [comment]
This is actually fixed via https://github.com/Microsoft/vscode/commit/479855f9cd756a2b16fcf42177cf54f86c2a0dfb since it sets the placeholder text color.

**Comment by misolori** [comment]
https://github.com/Microsoft/vscode/commit/479855f9cd756a2b16fcf42177cf54f86c2a0dfb is now in `master` for our default [Dark](https://github.com/Microsoft/vscode/blob/2eb31c71704b3505d4b17aaa7e707d5d8651e901/extensions/theme-defaults/themes/dark_defaults.json#L14) and [Light](https://github.com/Microsoft/vscode/blob/2eb31c71704b3505d4b17aaa7e707d5d8651e901/extensions/theme-defaults/themes/light_defaults.json#L15) themes
